src/components/Card.js

```diff
@@ -32,18 +48,24 @@ export default class Card {
       ".card__description-text"
     );
 
-    cardImageEl.src = this.link; //adds card image
-    cardImageEl.alt = this.name; //adds card alt text
-    cardNameTextEl.textContent = this.name; //Description
+    cardImageEl.src = this._link; //adds card image
+    cardImageEl.alt = this._name; //adds card alt text
+    cardNameTextEl.textContent = this._name; //Description
   }
 
   _handleLikeButton() {
+    if(this.isLiked)
+    {api.unlikeCard(this.id).then(json => this.liked = false)}
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere**